docs/application/dotnet/guides/opentk/quickstart.md

```diff
@@ -0,0 +1,266 @@
+            }
+            return shader;
+        }
+
+        static void Main(string[] args)
+        {
+            using (OpenTKApp app = new OpenTKApp() { GLMajor = 2, GLMinor = 0 })
+            {
+                app.Run(args);
+            }
+        }
+    }
+}
+```
+
+- `OpenTKApp` is derived from `TizenGameApplication`. 
+    `TizenGameApplication` creates OpenGL&reg; ES running environment, which includes window, OpenGL&reg; ES context, and so on.
+    `TizenGameApplication` inherits from `Tizen.Applications.CoreUIApplication`, and you can override the application related methods including `OnCreate`, `OnPause`, `OnResume`, `OnAppControlReceived`, `OnTerminate`, and so on. For more information about these methods, see [Applications](../applications/ui-app.md).
+- `InitShader` : `InitShader` is an event handler of `Window.Load` event. It initializes shaders of triangle in this sample.
+- `RenderTriangle` : `RenderTriangle` is an event handler of `Window.RenderFrame` event. It draws the triangle in this sample.
+- `app.Run()` : It runs the main loop of OpenTK app.
+
+### Build and Launch your Application
+
+1.  Building the solution:
+    - In the Visual Studio menu, select **Build** > **Build Solution**.
+    - In the Solution Explorer view, right-click the solution name and select **Build**.
+
+2.  Launching Tizen Emulator:
+
+    - In the Visual Studio menu, select **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
+    - Alternatively, click **Launch Tizen Emulator** in the Visual Studio toolbar to launch the Tizen Emulator Manager:
+
+        ![launch_emulator1](media/tizen_project_wizard_capture_template3.png)
+
+    - In the Emulator Manager, select an emulator from the list, and click **Launch**:
+        ![launch_emulator2](media/tizen_project_wizard_capture_template4.png)
+    - You must update the following emulator images to use the latest version (Tizen 5.0).
+
+3.  After you launch an emulator instance, you can select the target from the drop-down list to change the deployment target in the Visual Studio toolbar:                                                          
+        ![VS_Toolbar2](media/tizen_project_wizard_capture_template6.png)
+        
+    You can deploy your OpenTK app to the target, using the following::
+
+     1. Press the **F5** or **Ctrl+F5** key.
+     2. In the Visual Studio menu, Select **Debug > Start Debugging** or **Start Without Debugging**.
+     3. On the Visual Studio toolbar, click an emulator instance:                                               
+     ![VS_Toolbar](media/tizen_project_wizard_capture_template5.png)
+                         
+4.  If the deployment is successful, then the application is launched on the target emulator.
+The following are snapshots of launching this OpenTK app on mobile and TV emulators:
+
+    Figure: OpenTK app on TV emulator:                             

```

Comment:
Global: Please use figure caption is the following format:
**Figure: OpenTK app on TV emulator**
Note: Please don't use colon at the end of the figure caption